### ISSUE TITLE
Leading space in to_string(index=False)

### ISSUE THREAD

**Issue by varun-kr** [pydata/pandas#11833]
#11750
```python 
In [2]: df = pd.DataFrame({'a':range(5)})

In [3]: df.to_string()
Out[3]: '   a\n0  0\n1  1\n2  2\n3  3\n4  4'

In [4]: df.to_string(index=False)
Out[4]: ' a\n 0\n 1\n 2\n 3\n 4'
```
What is the use of a leading space when index = False ?

**Comment by jreback** [comment]
agreed (though in a test prob want to use a slightly larger range so we have 2 digit numbers (the result is the same though, still 1 leading space)

**Comment by Vutsuak16** [comment]
@jreback just have a look at my commit :)

**Comment by jreback** [comment]
closed by #11942
